implementation/src/test/java/io/smallrye/mutiny/groups/UniMemoizeTest.java

```diff
@@ -222,11 +255,12 @@ public void assertCancellingImmediately() {
         sub2.assertNotTerminated();
     }
 
-    @Test
+    @RepeatedTest(10)
+    @DisplayName("Test uni.memoize().forever() for race conditions on subscription and cancellation")
     public void testSubscribersRace() {
         for (int i = 0; i < 2000; i++) {
             Flowable<Integer> flowable = Flowable.just(1, 2, 3);
```

Comment:
I believe we can replace all the Flowable using Multi. This was developed before Multi was a thing...